tests/test_sim_pipeline.py

```diff
@@ -2,23 +2,55 @@
 
 """Tests for `sim_pipeline` package."""
 
-import pytest
+import unittest
```

Comment:
can you write the test functions in files and folders that mimic the sim-pipeline file structure such that we all know where exactly to look for a test function for a specific definition

Reply:
Updated